source/includes/_endpoint-dataset.md

```diff
@@ -777,12 +777,13 @@ The following rules apply for all formats:
 
 Some format-specific properties and options:
 
-Format    | Attribute           | Description                                                           | Example
---------- | --------------------| --------------------------------------------------------------------- | --------------------------
-csv       | use_category_ids    | instead of category names export the fields as their numeric ids      | {"use_category_ids": true}
-spss      | var_label_field     | Use the variable's name/description as SPSS variable label            | {"var_label_field": "name"}
-spss      | prefix_subvariables | When True, will only export subvariables and not group them as arrays | {"prefix_subvariables": true}
-all       | include_personal    | Will include the user's personal variables on the exported file       | {"include_personal": true}
+Format    | Attribute           | Description                                                                         | Default
+--------- | --------------------| ----------------------------------------------------------------------------------- | --------------------------
+csv       | use_category_ids    | Instead of category names export the fields as their numeric ids                    | {"use_category_ids": false}
+csv       | missing_values      | Which value to use for printing missing values; allowed "dot", "reason", "blank"    | {"missing_values": "reason}
```

Comment:
"" is too falsey, I prefer the more explicit option. Cannot be confused with "forgot value" case.